graphs/possible_bipartition.py

```diff
@@ -1,12 +1,72 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
+from pickle import FALSE 
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        
+        Time Complexity: O(1) to access the adjacency list, o(1) to iterate through the dog_dict hash map used to 
+        organize the dislikes data into a more accessible format(w/out empty indices), o(n) to iterate through each value list of the 
+        dictionary, which is dependent on the size of the input. 
+        
+        Space Complexity: O(n) for new varaibles created (visited list, dictionary) b/c the space occupied is dependent on the input. 
+        O(E) for the adjacency list because each node in the dislikes array has a list of edges. 
     """
```

Comment:
👍  Nice DFS Solution.

However the time complexity would be O(VE) where V is the number of vertices and E the number of edges.